lua_examples/telnet/README.md

```diff
@@ -0,0 +1,79 @@
+# Telnet Module
+
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2014-12-22 | [Zeroday](https://github.com/funshine) |  [Terry Ellison](https://github.com/TerryE) | [simple_telnet.lua](./simple_telnet.lua) |
+| 2018-05-24 | [Terry Ellison](https://github.com/TerryE) |  [Terry Ellison](https://github.com/TerryE) | [telnet.lua](./telnet.lua) |
+
+
+The Lua telnet example previously provided in our distro has been moved to this 
+file `simple_telnet.lua` in this folder. This README discusses the version complex 
+implementation at the Lua module `telnet.lua`.  The main reason for this complex
+alternative is that a single Lua command can produce a LOT of output, and the 
+telnet server has to work within four constraints:
+
+-  The SDK rules are that you can only issue one send per task invocation, so any 
```

Comment:
I'm not convinced that this is true. I think that the real rule is that you can call send as often as you like, but if it fails, then you must retry that send later (after the on:sent callback is called). 

I might try and hack on the net module to make this happen automatically.

Reply:
This was a strict rule, but you are probably correct in that open LwIP has probably relaxed this, however if you do a `print(a,b,c,d)` then this sends 5 separate fields and stick something like this in a for loop then you reboot the ESP with the old telnet. The new one is far more robust and you can happily print out 10k.